tests/sqllogictests/suites/base/05_ddl/05_0035_ddl_set_settings.test

```diff
@@ -0,0 +1,35 @@
+#need onlyif mysql tag, because http test can not keep a session
+query T
+show settings like 'timestamp_output_format';
+----
+timestamp_output_format '%Y-%m-%d %H:%M:%S%.6f' '%Y-%m-%d %H:%M:%S%.6f' ["%Y-%m-%d %H:%M:%S%.3f", "%Y-%m-%d %H:%M:%S%.6f"] SESSION Specifies the display format for the TIMESTAMP data type alias. Available values include '%Y-%m-%d %H:%M:%S%.6f' and '%Y-%m-%d %H:%M:%S%.3f'. String
+
+onlyif mysql
```

Comment:
why need `onlyif mysql`?

Reply:
Now sql logic test http handwriting can not store session settings .

But if set global in test file, it maybe cause some flaky test.